sensor_board/common/pinav_parser.c

```diff
@@ -0,0 +1,266 @@
+#include "pinav_parser.h"
+#include <string.h>
+#include <stdio.h>	// @TODO: remove when not needed for debugging
+
+typedef enum generic_status { FAIL = 0, SUCCESS = 1 } generic_status_t;
```

Comment:
We may want to have a common definition for this somewhere, it seems generally useful.